src/components/DpUploadFiles/DpUpload.vue

```diff
@@ -242,7 +246,7 @@ export default {
 
     this.uppy.on('upload-error', (file, error, response) => {
       console.error(error)
-      dplan.notify.error(Translator.trans('error.fileupload'))
+      dplan.notify.error(this.uppyTranslations.errorFileUpload)
```

Comment:
Aren't the translation keys now in `this.uppyTranslations.strings`?

Reply:
Yes, it's there: `errorFileUpload: german.validation.error.fileUpload,`

Reply:
https://github.com/demos-europe/demosplan-ui/pull/918#discussion_r1672430610

Reply:
My question is, shouldn't it be
`dplan.notify.error(this.uppyTranslations.strings.errorFileUpload)`
instead of
`dplan.notify.error(this.uppyTranslations.errorFileUpload)`?

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/918/commits/9e7a0985c4d3ef03a8cfd87d6b7667b363137690